test/libethereum/StateTestsFiller/EIP150/Homestead/stSystemOperationsTestFiller.json

```diff
@@ -895,15 +903,15 @@
             "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
                 "nonce" : "0",
                 "storage" : {
-                    "0x" : "0x80"
+                    "0x" : "0x00"
                 }
             }
         },
         "pre" : {
             "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
                 "balance" : "1000000000000000000",
                 "nonce" : "0",
-                "code" : "{ (MSTORE 0 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff) (MSTORE 32 0xaaffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffaa ) [[ 0 ]] (CALLSTATELESS 500 0x945304eb96065b2a98b57a48a06ae28d285a71b5 23 0 64 0 2 ) }",
```

Comment:
The test is still called `callstateless`. What kind of test is this supposed to be? callcode?